test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/SchedulesExpiryDuringReconnect.java

```diff
@@ -92,7 +97,8 @@ private HapiApiSpec expireSchedulesDuringReconnect() {
 								.adminKey(DEFAULT_PAYER)
 								.logging()
 								.advertisingCreation()
-								.savingExpectedScheduledTxnId()
+								.savingExpectedScheduledTxnId(),
+						sleepFor(Duration.ofSeconds(5).toMillis())
```

Comment:
nit: I guess this sleep is to obtain a good timing for reconnect. Should add a comment to explain it.

Reply:
Oh yes .. this was added for debug purpose. Will remove it now. Thanks @qnswirlds !